aspnet/mvc/controllers/areas.rst

```diff
@@ -35,19 +43,29 @@ Instead of having all of the controllers located under the Controllers parent di
 
           - Index.cshtml
 
-Looking at the preceding directory hierarchy example, there are a few guidelines to keep in mind when defining areas:
+When MVC tries to render a view in an Area, by default, it tries to look in the following locations:
+
+.. code-block:: txt
+
+  /Areas/<Area-Name>/Views/<Controller-Name>/<Action-Name>.cshtml
```

Comment:
Do users really need the leading /Area in the URL? that seems like unnecessary friction.